cmd/tidb-dashboard/main.go

```diff
@@ -80,11 +82,28 @@ func NewCLIConfig() *DashboardCLIConfig {
 	flag.Int64Var(&cfg.KVFileStartTime, "keyviz-file-start", 0, "(debug) start time for file range in file mode")
 	flag.Int64Var(&cfg.KVFileEndTime, "keyviz-file-end", 0, "(debug) end time for file range in file mode")
 
+	caPath := flag.String("cacert", "", "path of file that contains list of trusted SSL CAs.")
```

Comment:
maybe we can call it `cluster-ca`

Reply:
Change to `cluster-ca`, `cluster-cert` and `cluster-key`